css-flexbox-photo-gallery/css/style30.css

```diff
@@ -0,0 +1,41 @@
+* {
```

Comment:
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: Arial;
  /*
  Apply a light gray background to the page by setting the `background` to `rgb(224, 221, 221);`
  */
  
}